.github/workflows/latest_release.yaml

```diff
@@ -29,15 +29,17 @@ jobs:
       continue-on-error: true
       run: |
         gh release delete latest --repo $GITHUB_REPOSITORY
+        sleep 3
```

Comment:
should we start with sleep only on L32 first ?

Reply:
also would be good to add a comment explaining the WHY

Reply:
I think all sleeps matters. GH APIs take sometime to make effect, we should make sure before `Recreate the Latest Release and Tag` tag and release all get deleted.

And after `Recreate the Latest Release and Tag`, we need to sleep some time to wait new tag and release are created before checking it.

Reply:
Okay can you add this info as comments above each sleep ?

Reply:
Yes.